app/ptxboa_functions.py

```diff
@@ -109,6 +109,12 @@ def calculate_results_list(
         res_list.append(res_single)
     res_details = pd.concat(res_list)
 
+    # drop Green Iron if comparing chains (because it is not an energy carrier)
```

Comment:
you could also filter the parameter list before the calculation to gain performance: 
```python

    if parameter_to_change == "chain":
        parameter_list = parameter_list[~parameter_list.str.startswith("Green Iron")]
```

Reply:
good idea, thanks!